core/src/main/java/org/aerogear/mobile/core/http/OkHttpCertificatePinningParser.java

```diff
@@ -0,0 +1,26 @@
+package org.aerogear.mobile.core.http;
+
+import java.util.List;
+
+import org.aerogear.mobile.core.configuration.https.CertificatePinningEntry;
+
+import okhttp3.CertificatePinner;
+
+public class OkHttpCertificatePinningParser implements HttpCertificatePinningParser {
+
+    private final List<CertificatePinningEntry> pinningConfig;
+
+    public OkHttpCertificatePinningParser(final List<CertificatePinningEntry> pinningConfig) {
+        this.pinningConfig = pinningConfig;
+    }
+
+    @Override
+    public CertificatePinner parse() {
+        CertificatePinner.Builder certPinnerBuilder = new CertificatePinner.Builder();
+        for (CertificatePinningEntry pinningEntry : pinningConfig) {
+            String fullPinningHash = "sha256/" + pinningEntry.getCertificateHash();
```

Comment:
is there a reason that this prefix is not part of the hash value? I think it should be part of the hash value, otherwise it is possible that hash generation using a different alg, but it is not changed here.